### PR TITLE
New tags handling, issue #22

### PR DESCRIPTION
**Pull request by libremente**
## Title
This PR changes the whole tags handling tackling issue #28.
## Content
-  2 new files are introduced: `categories-list.rst` and `scope-list.rst`.
-  `freeTags` are removed to allow for better indexing etc.
-  the `tags` key has been replaced with the `category` one where each entry must be a valid value in `categories-list`
-  the `standaloneType` key is introduced in order to help defining the type of software
-  the `intendedAudience/scope` key is introduced as an optional key where each entry must be a valid value in `scope-list`

This is a breaking change.

## Review

- [x] Ensure your files are written following RST specs (*not MD!*)
- [x] Italian version
- [ ] English version
- [x] Examples
- [x] Ask for review

### CODE REVIEW

File: docs/it/categories-list.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+CAD
+call-center-management
+cloud-management
+collaboration
+communications
+compliance-management
+contact-management
+content-management
+CRM
+customer-service-&-support
```
Comment (libremente) [inline docs/it/categories-list.rst]:
```suggestion
customer-service-and-support
```

File: docs/it/categories-list.rst [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+e-signature
+email-management
+email-marketing
+employee-management
+enterprise-project-management
+enterprise-social-networking
+ERP
+event-management
+facility-management
+feedback-&-reviews-management
```
Comment (libremente) [inline docs/it/categories-list.rst]:
```suggestion
feedback-and-reviews-management
```

File: docs/it/categories-list.rst [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+financial-reporting
+fleet-management
+fundraising
+gamification
+geographic-information-systems---gis
```
Comment (libremente) [inline docs/it/categories-list.rst]:
```suggestion
geographic-information-systems
```

File: docs/it/categories-list.rst [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+identity-management
+instant-messaging
+inventory-management
+it-asset-management
+it-development
+it-management
+it-security
+it-service-management
+knowledge-management
+learning-management-system---lms
```
Comment (libremente) [inline docs/it/categories-list.rst]:
```suggestion
learning-management-system
```

File: docs/it/categories-list.rst [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+application-development
+appointment-scheduling
+architecture
+backup
+billing-and-invoicing
+blog
+budgeting
+business-intelligence
+business-process-management
+CAD
```
Comment (libremente) [inline docs/it/categories-list.rst]: lowercase

File: docs/it/scope-list.rst [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+| science-and-technology                   |                           |
++------------------------------------------+---------------------------+
+| society	                           |                           |
++------------------------------------------+---------------------------+
+| sport                                    |                           |
++------------------------------------------+---------------------------+
+| transportation                           |                           |
++------------------------------------------+---------------------------+
+| tourism                                  |                           |
++------------------------------------------+---------------------------+
```
Comment (alranel) [inline docs/it/scope-list.rst]: Aggiungere:
* `welfare`
* `emergency-services`
* `security`
* `agriculture`

File: docs/it/example/publiccode.yml [hunk 7]
```diff
@@ -20,10 +19,9 @@ platforms:
   - android
   - ios
 
-tags:
+category:
```
Comment (libremente) [inline docs/it/example/publiccode.yml]:
```suggestion
categories:
```

File: docs/en/example/publiccode.yml [hunk 8]
```diff
@@ -20,10 +19,9 @@ platforms:
   - android
   - ios
 
-tags:
+category:
```
Comment (libremente) [inline docs/en/example/publiccode.yml]:
```suggestion
categories:
```

File: docs/it/schema.rst [hunk 9]
```diff
@@ -208,21 +208,18 @@ Se possibile, usare i valori predefiniti. Se il software gira su una
 piattaforma per la quale un valore predefinito non è disponibile, un
 diverso valore può essere usato.
 
-Chiave ``tags``
-~~~~~~~~~~~~~~~
+Chiave ``category``
```
Comment (libremente) [inline docs/it/schema.rst]:
```suggestion
Chiave ``categories``
```

File: docs/it/schema.rst [hunk 10]
```diff
@@ -277,16 +274,22 @@ Chiave ``softwareType``
 
 -  Tipo: stringa
 -  Presenza: obbligatoria
--  Valori permessi: ``standalone``, ``addon``, ``library``,
-   ``configurationFiles``
+-  Valori permessi: ``standalone/mobile``, ``standalone/iot``,
+  ``standalon/desktop``, ``standalon/web``, ``standalone/backend``,
```
Comment (libremente) [inline docs/it/schema.rst]:
```suggestion
  ``standalone/desktop``, ``standalone/web``, ``standalone/backend``,
```

**Pull request merged**